tests/conftest.py

```diff
@@ -7,14 +7,20 @@
 
 CCDS_ROOT = Path(__file__).parents[1].resolve()
 
-args = {
+base_args = {
     "project_name": "NestaTestCookie".lower(),
     "author_name": "Nesta",
     "repo_name": "nestatestcookie",
+    "openness": "public",
 }
+args_list = [
+    {**base_args, "venv_type": "uv"},
+    {**base_args, "venv_type": "venv"},
+    {**base_args, "venv_type": "conda"},
+]
```

Comment:
Good sense to paramaterise like this :)

Reply:
You saying this made me realise I could have also just done a comprehension which i think is even neater.

Reply:
It is, I might go further and rename `args_list` to `test_params`